src/test/a11y/results.js

```diff
@@ -8,7 +8,7 @@ const handleTestResults = results => {
 
   if (resultsWithIssues.length) {
     console.error('Accessibility issues have been found:')
-    resultsWithIssues.forEach(console.error)
+    console.error(JSON.stringify(resultsWithIssues, null, 2))
```

Comment:
What do the `null` and `2` values mean for this method call?

Reply:
Should we add a comment that this is for pretty printing? null could be replaced with _ to show that we actually don't care about this parameter.

Reply:
we can't replace numm with `_` as that only works when defining a function, not calling it. I have added a comment.